cmake/modules/Findpmem.cmake

```diff
@@ -20,7 +20,7 @@ foreach(component ${pmem_FIND_COMPONENTS})
   if(pmem_FIND_VERSION_EXACT)
     set(pkg_module_spec "lib${component}=${pmem_FIND_VERSION}")
   elseif(pmem_FIND_VERSION)
-    set(pkg_module_spec "lib${component}>=${pmem_FIND_VERSION}")
+    set(pkg_module_spec "lib${component}<=${pmem_FIND_VERSION}")
```

Comment:
i don't understand this change. per https://cmake.org/cmake/help/latest/module/FindPkgConfig.html

> `foo>=3.1` matches any version from 3.1 or later.

Reply:
> i don't understand this change. per https://cmake.org/cmake/help/latest/module/FindPkgConfig.html
> 
> > `foo>=3.1` matches any version from 3.1 or later.

pmem_FIND_VERSION is system version(1.6), so it should >= required verison. Corresponding to this example, foo means pmem_FIND_VERSION,  3.1 means our required version.

Reply:
no, `pmem_FIND_VERSION` in this case is the requested version string, per https://cmake.org/cmake/help/latest/command/find_package.html#version-selection

> `PACKAGE_FIND_VERSION`
>
>        Full requested version string

i think the correct fix should be #41022

Reply:
Thanks for solving the problem and explaining, I will withdraw this commit.